src/synopsys-action/synopsys-bridge.ts

```diff
@@ -185,7 +185,9 @@ export class SynopsysBridge {
     let latestVersion = '0.0.0'
 
     for (const version of versionArray) {
-      if (version > latestVersion) {
+      const latestVersionParts = latestVersion.split('.')
+      const versionParts = version.split('.')
+      if ((parseInt(versionParts[0]) >= parseInt(latestVersionParts[0]) && parseInt(versionParts[1]) >= parseInt(latestVersionParts[1]) && parseInt(versionParts[2]) >= parseInt(latestVersionParts[2])) || (parseInt(versionParts[0]) >= parseInt(latestVersionParts[0]) && parseInt(versionParts[1]) >= parseInt(latestVersionParts[1])) || parseInt(versionParts[0]) >= parseInt(latestVersionParts[0])) {
```

Comment:
for readability can we break the condition as pathcversioncheck, majorversion n minor version checks? also too any condition in one line

Reply:
Here we are checking for latest version considering all the three versioning system. If we start putting for different version check, we will have to introduce 3 conditions and in each iteration any type of version can be picked, depending upon the last version in the array. Let me know, if it is okay.

Otherwise we will try to go with 3 conditions

Reply:
ok can we atleast keep the condition in multi line , it's difficult to read the conditions

Reply:
Done